htdocs/api/index.php

```diff
@@ -434,4 +434,17 @@
 	echo $result;
 }
 
+// Now flush output buffers so that response data is sent to request client
+ob_end_flush();
+ob_flush();
+flush();
```

Comment:
We did a ob_end_flush, so it means we have returned the output to the client and we now have time to do all the rest. So why doing a flush here. It forces to make IO when they can be delayed and managed by the system when he can.

Reply:
Agreed